content/en/blog/2021/istio-community-meetup-china/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "The first local Istio meetup is here!"
+description: "Join Istio community experts in China"
```

Comment:
add `.` in the end will fix the lint error:  
description: "Join Istio community experts in China."